omop_file_validator.py

```diff
@@ -251,7 +249,7 @@ def run_checks(file_path, f):
                     submission_column_type = df[submission_column].dtype
 
                     # If all empty don't do type check
-                    if submission_column_type is not None:
```

Comment:
I'm not sure the dtype property of a series can be None.

Reply:
You're right, the dtype cannot be None.

[List of dtypes here](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.dtypes.html)